sdk/ai/azopenai/client_embeddings_unit_test.go

```diff
@@ -21,6 +21,13 @@ func TestDeserializeEmbeddingsArray(t *testing.T) {
 		require.Equal(t, rawJSON[1:len(rawJSON)-1], ei.EmbeddingBase64)
 	})
 
```

Comment:
Let's split this into two PRs. Normally, it's not a big deal, but it can help with keeping the PRs focused.